commec/regulation/containers.py

```diff
@@ -0,0 +1,383 @@
+# Copyright (c) 2021-2024 International Biosecurity and Biosafety Initiative for Science
+
+"""
+Defines Containers to show default layouts for regulation data.
+What is the shape of the regulation data, and where to store it.
+
+    RegulationContainer : 
+        A standard output that may be associated with a hit.
+        Contains list, region, and level information.
+    RegulationLevel :
+        Determines valid level information strings.
+    TaxidRegulationContainer :
+        Storage of all regulation data for a given taxid from a given list.
+
+Also defines the storage data for 
+    * regulation lists, 
+    * per taxid regulation information per list, 
+    * and child to parent taxid mapping.
+"""
+import logging
+import re
+from dataclasses import dataclass, field, fields
+from typing import Optional
+from enum import StrEnum
+import pandas as pd
+
+logger = logging.getLogger(__name__)
+
+class AccessionFormat(StrEnum):
+    """
+    Supported hashable accesion formats.
+    """
+    TAXID = "TaxonomyID"
```

Comment:
Do you know the most standard names for these? We're definitely only talking about the NCBI taxonomy database (that is what we ask people to download) - looking at [this page](https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode=Info&id=186536&lvl=3&lin=f&keep=1&srchmode=5&unlock) maybe the term we use in code is "taxid" and in text is "NCBI Taxonomy ID"?